Xamarin.Forms.Platform.UAP/VisualElementRenderer.cs

```diff
@@ -155,7 +155,7 @@ public void SetElement(VisualElement element)
 
 			OnElementChanged(new ElementChangedEventArgs<TElement>(oldElement, Element));
 
-			if (_control != null && this is ITabStopOnDescendants)
+			if (_control != null && (this is ITabStopOnDescendants || this is ScrollViewRenderer))
```

Comment:
Is there a reason why the `ScrollViewRenderer` is _NOT_ a `ITabStopOnDescendants`? I see search boxes and pickers are all `ITabStopOnDescendants`, so why not scrollviews?

Reply:
`ITabStopOnDescendants` does not allow the focus to stop on the current control and **allow focus to all Descendants**. The last point we do not need, because it will turn on/off TapStops for all items inside in `ScrollView`.

Reply:
Would it maybe be better to have a new interface? What happens if we create a HorizontalScrollView and a ZoomingView? Will this if statement get bigger?

Reply:
done